### ISSUE TITLE
Usage of alpha in attribute loss

### ISSUE THREAD

**Issue by Demirrr** [dice-group/LitCQD#1]
`alpha * (l_fit+l_reg) + alpha * (attr_loss + attr_reg)`
see [here](https://github.com/dice-group/LitCQD/blob/698159eb66188a56ddc4c124ca969f48018b8a75/models/CQDComplExA.py#L90)
should be changed to
`(l_fit+l_reg) + (1 - alpha) * (attr_loss + attr_reg)`

Please also add the following assertions somewhere before the training :)
`assert 1>=alpha>=0.0`

Please also update `[this loss function](https://github.com/dice-group/LitCQD/blob/698159eb66188a56ddc4c124ca969f48018b8a75/models/CQDComplExAD.py#L44)`

`alpha * (l_fit + l_reg) + alpha * (attr_loss + attr_reg) + alpha * (desc_loss + desc_reg)`

`l_fit + l_reg + (alpha * (attr_loss + attr_reg)) + beta * (desc_loss + desc_reg))`

where beta is also a hyperparameter parameter.
